src/main/java/org/gridsuite/modification/server/NetworkModificationController.java

```diff
@@ -88,14 +88,23 @@ public ResponseEntity<Flux<ModificationInfos>> getModification(@Parameter(descri
     }
 
     @PutMapping(value = "/groups/{groupUuid}/modifications/move", produces = MediaType.APPLICATION_JSON_VALUE)
-    @Operation(summary = "Get list of modifications metadata of a group")
+    @Operation(summary = "Move some network modifications before another one, or at the end of the list")
     @ApiResponse(responseCode = "200", description = "List of modifications of the group")
     public ResponseEntity<Mono<Void>> moveModifications(@Parameter(description = "Group UUID") @PathVariable("groupUuid") UUID groupUuid,
                                                   @Parameter(description = "before") @RequestParam(value = "before", required = false) UUID before,
                                                   @Parameter(description = "modification to moves", required = true) @RequestParam(value = "modificationsToMove", required = false) List<UUID> modificationsToMove) {
         return ResponseEntity.ok().body(networkModificationService.moveModifications(groupUuid, before, modificationsToMove));
     }
 
+    @PutMapping(value = "/groups/{groupUuid}/modifications/duplicate")
```

Comment:
Not compliant with the best pratices for gridsuite API rest
Remove `/modifications/duplicate`

Reply:
Add a request param to type the modification

Reply:
yes, a param named "action" has been added to perform either a MOVE or a DUPLICATE operation.
As a consequence, existing /modifications/move has also been remove.